packages/date-picker/src/panel/date.vue

```diff
@@ -46,6 +48,7 @@
             </span>
           </div>
           <div
+            tabindex="-1"
```

Comment:
This is a general question, but does the tab index on this container prevent tabbing to any of its children?

Reply:
Ha, no it isn't doing a dang thing. I tested it to see if it would work on the container and it appeared to but only because we  don't need `tabindex=1` on anything since the `preventDefault` makes it impossible to tab to anything in the calendar. I've removed it.